### DISCUSSION TITLE
Exploit section added to vulnerability scanning

### DISCUSSION THREAD

**Discussion by Marcopolo0** [aquasecurity/trivy#5967] [Ideas]
### Description

If it's feasible, could you add a section titled "Exploits" where you provide links to publicly available exploits for each CVE? Currently, there is a "References" section, but having an additional section dedicated to storing links to publicly available exploits corresponding to findings in vulnerability scans would be extremely helpful.

### Target

Filesystem

### Scanner

Vulnerability

**Comment by itaysk** [comment]
thanks for the suggestion. where do you suppose that content for exploits section will come from?

**Reply by Marcopolo0** [reply]
A possible option could be browsing the NVD database like in the link: https://nvd.nist.gov/vuln/detail/CVE-2021-44228, in the context of the occurrence of the word "exploit", or grepping through databases: GitHub, Packet Storm, ExploitDB, and returning results similar to a resources section where there are links to vulnerabilities.
